src/view/LoginView.java

```diff
@@ -66,13 +74,13 @@ public void actionPerformed(ActionEvent evt) {
           }
         });
 
-    cancel.addActionListener(this);
-
-    switchToSignup.addActionListener(
+    cancel.addActionListener(
```

Comment:
Remember to resolve the cancel issue when this is merged.

Reply:
I think I forgot, which cancel issue are you talking about here? This PR makes cancelling go back to the welcome view

Reply:
Issue #2

Reply:
Aha, there's a neat trick with GitHub, I can edit the PR description to say `Fixes #2` and then merging this will close that issue